frontend/src/components/HomePageDesktopFilter/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React, {useEffect} from 'react';
+import styled from 'styled-components';
+import {
+    Tab,
+    Tabs,
+    Hidden,
+} from '@material-ui/core'
+
+const DeskTopFilters = styled(Tabs)`
+`;
+
+const HomePageDesktopFilter = () => {
```

Comment:
It might be worth considering combining the HomePageDesktopFilter and the HomePageMobileFilter into a single component to get rid of some code redundancy (but don't worry about it if you don't want to lol, as long as it works I'm fine with either 😅  )